src/plugins/ui_actions/public/context_menu/build_eui_context_menu_panels.tsx

```diff
@@ -23,7 +22,7 @@ export const txtMore = i18n.translate('uiActions.actionPanel.more', {
 });
 
 interface ActionWithContext<Context extends object = object> {
-  action: Action<Context>;
+  action: Action | ActionInternal<ActionDefinition<Context>>;
```

Comment:
should it at least be? 

```suggestion
  action: Action<Context> | ActionInternal<ActionDefinition<Context>>;
```